gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/DefaultTokenAuthorityService.java

```diff
@@ -104,26 +103,15 @@ public void setAliasService(AliasService as) {
 
   @Override
   public JWT issueToken(JWTokenAttributes jwtAttributes) throws TokenServiceException {
-    String[] claimArray = new String[6];
```

Comment:
JWTokenAttributes has replaced the claimsArray throughout?

Reply:
Yes, I discussed it with @smolnar82 and we decided to remove the claimsArray, and use the JWTTokenAttributes instead.